driver-core/src/main/java/com/datastax/driver/core/AbstractTableMetadata.java

```diff
@@ -258,14 +258,19 @@ protected StringBuilder appendOptions(StringBuilder sb, boolean formatted) {
     sb.append("WITH ");
     if (options.isCompactStorage()) and(sb.append("COMPACT STORAGE"), formatted);
     if (!clusteringOrder.isEmpty()) and(appendClusteringOrder(sb), formatted);
-    sb.append("read_repair_chance = ").append(options.getReadRepairChance());
-    and(sb, formatted)
-        .append("dclocal_read_repair_chance = ")
-        .append(options.getLocalReadRepairChance());
+    if (cassandraVersion.getMajor() < 4)
+      sb.append("read_repair_chance = ").append(options.getReadRepairChance());
+    else sb.append("read_repair = 'BLOCKING'");
+    if (cassandraVersion.getMajor() < 4)
+      and(sb, formatted)
+          .append("dclocal_read_repair_chance = ")
+          .append(options.getLocalReadRepairChance());
```

Comment:
I have created https://datastax-oss.atlassian.net/browse/JAVA-2730 for addressing this.